src/pages/about/_our-locations.js

```diff
@@ -57,6 +57,7 @@ const locations = [
     {
         name: localize('Dubai,'),
```

Comment:
remove the comma as well?
```suggestion
        name: localize('Dubai'),
```

Reply:
dubai is part of UAE, same like other place, we add comma for city